.github/workflows/pay-api-ci.yml

```diff
@@ -1,47 +1,69 @@
-name: Pay API CI
+name: Report API CI
 
 on:
   pull_request:
     branches:
-      - development
+      - main
     paths:
-      - "pay-api/**"
+      - "report-api/**"
```

Comment:
Isn't this CI for pay-api? all references are for report

Reply:
Messed up by copy & paste. Fixed.